src/classes/STG_PanelRDBatch_CTRL.cls

```diff
@@ -52,6 +52,15 @@ public with sharing class STG_PanelRDBatch_CTRL extends STG_Panel {
     */
     public Boolean isRunningBatch { get; set; }
 
+    /*********************************************************************************************************
+    * Description The Recurring Donation Batch Introduction Text
+    */
+    public String getPageIntro() {
+        return (RD2_EnablementService.isRecurringDonations2Enabled)
```

Comment:
The `STG_Panel` contains `isRD2Enabled` property that can be used  instead, for example:
 ```
 return isRD2Enabled
    ? System.Label.stgHelpEnhancedRDBatch
    : System.Label.stgHelpRDBatch;
```